pallets/stream-payment/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+//! Runtime API for Stream Payment pallet
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
```

Comment:
```suggestion
```

Reply:
How do you import `String` in `no_std` without `alloc`?

Reply:
Sorry, I though `use alloc` already imported the crate. Maybe we can import `String` from `sp_std` instead?